pennylane/tape/tape.py

```diff
@@ -90,7 +90,7 @@ def get_active_tape():
     return QueuingContext.active_context()
 
 
-def expand_tape(tape, depth=1, stop_at=None, expand_measurements=False):
+def expand_tape(tape, depth=1, stop_at=None, expand_measurements=False, iterative_call=False):
```

Comment:
This new kwarg is used exclusively to make lower-level iterations of `expand_tape`, which calls itself, cheaper.

Reply:
Will add documentation etc once we settle for this approach